.github/workflows/ci.yml

```diff
@@ -521,9 +521,7 @@ jobs:
     steps:
     - name: Download XML reports
       if: success() || failure()
-      uses: actions/download-artifact@v3
-      with:
-        name: junit-xml
+      uses: actions/download-artifact@v4
```

Comment:
per the new docs, https://github.com/actions/download-artifact#inputs removing the name allows for downloading all the atifacts.

The docs are 20MB, but doing a couple of runs it looks like this is only 2seconds to download so this way seems the simplest approach.